pjlib-util/src/pjlib-util/resolver.c

```diff
@@ -542,7 +542,7 @@ PJ_DEF(pj_status_t) pj_dns_resolver_set_ns( pj_dns_resolver *resolver,
     pj_status_t status;
 
     PJ_ASSERT_RETURN(resolver && count && servers, PJ_EINVAL);
-    PJ_ASSERT_RETURN(count < PJ_DNS_RESOLVER_MAX_NS, PJ_EINVAL);
+    PJ_ASSERT_RETURN(count <= PJ_DNS_RESOLVER_MAX_NS, PJ_EINVAL);
```

Comment:
While here, perhaps also:
- update the error code to `PJ_ETOOMANY`?
- remote the redundant `count` check below?

Reply:
Yes, on release mode, PJ_ASSERT_RETURN will become regular error checking, so the condition `if (count > PJ_DNS_RESOLVER_MAX_NS)` will never be true and can be safely removed.

Reply:
Done and I also changed the return value to `PJ_ETOOMANY` which makes more sense because it is more specific here.